src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -17,6 +17,7 @@ abstract class Mollie_WC_Gateway_Abstract extends WC_Payment_Gateway
 
     const PAYMENT_METHOD_TYPE_PAYMENT = 'payment';
     const PAYMENT_METHOD_TYPE_ORDER = 'order';
+    const PAYMENT_METHOD_TYPE_MANDATE = 'mandate';
```

Comment:
This constant is never used. Do we need it?

Reply:
No, we don't, right. I will take it out.